quiz.rb

```diff
@@ -2,19 +2,55 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+  attr_reader :name , :breed , :age
+  def initialize (name, breed, age)
+    @name=name
+    @breed=breed
+    @age=age
+  end
+end
+
+
+
 # Question 2
 # Create a new Dog object.
 
+holly = Dog.new "Holly", "yellow lab", 4
```

Comment:
This is a mistake, but it's a mistake I really like. This line works just fine, and it works because Ruby doesn't actually read the parentheses `()` when you invoke a function. Sometimes, advanced Ruby programmers will even forgo the parentheses _and_ commas altogether!